pkg/detector/library/driver.go

```diff
@@ -135,19 +136,22 @@ func (d *Driver) DetectVulnerabilities(pkgID, pkgName, pkgVer string) ([]types.D
 }
 
 func createFixedVersions(advisory dbTypes.Advisory) string {
-	if len(advisory.PatchedVersions) != 0 {
-		return strings.Join(advisory.PatchedVersions, ", ")
-	}
-
 	var fixedVersions []string
-	for _, version := range advisory.VulnerableVersions {
-		for _, s := range strings.Split(version, ",") {
-			s = strings.TrimSpace(s)
-			if !strings.HasPrefix(s, "<=") && strings.HasPrefix(s, "<") {
-				s = strings.TrimPrefix(s, "<")
-				fixedVersions = append(fixedVersions, strings.TrimSpace(s))
+	if len(advisory.PatchedVersions) != 0 {
+		for _, version := range advisory.PatchedVersions {
+			fixedVersions = append(fixedVersions, version)
+		}
+	} else {
+		for _, version := range advisory.VulnerableVersions {
+			for _, s := range strings.Split(version, ",") {
+				s = strings.TrimSpace(s)
+				if !strings.HasPrefix(s, "<=") && strings.HasPrefix(s, "<") {
+					s = strings.TrimPrefix(s, "<")
+					fixedVersions = append(fixedVersions, strings.TrimSpace(s))
+				}
 			}
 		}
 	}
-	return strings.Join(fixedVersions, ", ")
+
+	return strings.Join(lo.Uniq(fixedVersions), ", ")
```

Comment:
If I remember correctly, `lo.Uniq` doesn't preserve order.
Can you test this and add another version range to the new test case to make sure we don't break the order of the versions (I mean keeping the versions in ascending order)?

Reply:
It does:
> The order of result values is determined by the order they occur in the array.